src/toil/batchSystems/mesos/executor.py

```diff
@@ -153,6 +203,47 @@ def sendUpdate(taskState, message=''):
         thread = threading.Thread(target=runTask)
         thread.start()
 
+    def _getSlaveState(self, slaveId):
+        """
+        Return a dict containing the information Mesos provides about slave with ID == slaveId
+
+        :param slaveId: Id of the slave of interest
+        """
+        # This doesn't need authentication
+        allSlaves = requests.get('http://' + self.masterAddress + '/master/slaves').json()['slaves']
```

Comment:
Uhhhhhh.... I'll use thsi for now but i'll try to think if a name that is not Infos lol!